conf/modules/cnvkit.config

```diff
@@ -25,6 +25,28 @@ process {
         ]
     }
 
+    withName: '.*:BAM_VARIANT_CALLING_GERMLINE_ALL:BAM_VARIANT_CALLING_CNVKIT:CNVKIT_CALL' {
+        ext.prefix = { "${cns.baseName}.germline.call" }
+        ext.when   = { params.tools && params.tools.split(',').contains('cnvkit') }
+        ext.args   = "--filter ci"
+        publishDir = [
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/variant_calling/cnvkit/${meta.id}/" },
+            pattern: "*{cns}"
+        ]
+    }
```

Comment:
at the moment, I did not find any recommendation from cnvkit for the call step as there is for the germline, see [here](https://cnvkit.readthedocs.io/en/stable/germline.html).
there are recommendations for [tumour analysis](https://cnvkit.readthedocs.io/en/stable/tumor.html) but is not related to the call module.
I would probably document this, rather than making a choice in the config.

Reply:
then how about we add:
```
    withName: '.*:BAM_VARIANT_CALLING_SOMATIC_ALL:BAM_VARIANT_CALLING_CNVKIT:CNVKIT_CALL' {
        ext.when   = false
    }
    withName: '.*:BAM_VARIANT_CALLING_TUMOR_ONLY_ALL:BAM_VARIANT_CALLING_CNVKIT:CNVKIT_CALL' {
        ext.when   = false
    }
```

To disable `CNVKIT_CALL` for somatic and tumor only?

Reply:
we can, but then you don't export in VCF (which is chained to CALL)?
if you don't add specific `ext.args`, you'd just call it but with defaults, which is (yes I understand :P) a bit of what batch does already.

Reply:
btw, the module `call` is important also for tumour analysis: you can make lots of assumptions and implement them in the call step. see [here](https://cnvkit.readthedocs.io/en/stable/pipeline.html#calling-methods).
the only issue is that we can't make any specific assumption ourselves, because they're very specific to the application of the user, and therefore would be better to leave it with defaults but let the user choose any option if they want to. no harm if they don't :)

Reply:
We don't export in VCF for now.
So if we wanted to add that, we'd need to do this instead:
```suggestion
    withName: '.*:BAM_VARIANT_CALLING_CNVKIT:CNVKIT_CALL' {
        ext.when   = { params.tools && params.tools.split(',').contains('cnvkit') }
        publishDir = [
            mode: params.publish_dir_mode,
            path: { "${params.outdir}/variant_calling/cnvkit/${meta.id}/" },
            pattern: "*{cns}"
        ]
    }
    withName: '.*:BAM_VARIANT_CALLING_GERMLINE_ALL:BAM_VARIANT_CALLING_CNVKIT:CNVKIT_CALL' {
        ext.prefix = { "${cns.baseName}.germline.call" }
        ext.args   = "--filter ci"
    }
    withName: '.*:BAM_VARIANT_CALLING_SOMATIC_ALL:BAM_VARIANT_CALLING_CNVKIT:CNVKIT_CALL' {
        ext.prefix = { "${cns.baseName}.somatic.call" }
    }
    withName: '.*:BAM_VARIANT_CALLING_TUMOR_ONLY_ALL:BAM_VARIANT_CALLING_CNVKIT:CNVKIT_CALL' {
        ext.prefix = { "${cns.baseName}.tumor_only.call" }
    }
```

Reply:
that would definitely be my choice :) also thanks :)